includes/definitions/intelliflash.yaml

```diff
@@ -0,0 +1,11 @@
+os: intelliflash
+text: 'Tegile IntelliFlash'
+type: storage
+icon: tegile
+over:
+    - { graph: device_storage, text: 'Storage Usage' }
+mib_dir:
+    - tegile
+discovery:
+    - sysDescr:
+      - Tegile IntelliFlash
```

Comment:
Looks like you can use `sysObjectID` instead for detection. From our stats `.1.3.6.1.4.1.43906.1` is unique to this device. sysObjectID is more stable for detection.